index.d.ts

```diff
@@ -1,5 +1,8 @@
 /// <reference types="node" />
```

Comment:
Unrelated to this PR, but this comment is the old way of doing this and shouldn't be included anymore I believe.

Reply:
Want me to yank while i'm in there?

Reply:
Totally your call.  I could be wrong, so you should set aside some time to make sure yanking it doesn't break anything if you do.  :)